osu.Game/Database/RealmAccess.cs

```diff
@@ -843,7 +846,7 @@ public IDisposable BlockAllOperations()
                     updateRealm = null;
                 }
 
-                Logger.Log(@"Blocking realm operations.", LoggingTarget.Database);
+                Logger.Log(@"Lock acquired for blocking operations", LoggingTarget.Database);
```

Comment:
nit: I assume by acquiring lock you mean the `realmRetrievalLock`? Since there's a timeout exception on compact failure below this log which states "Took too long to acquire lock", may need to reword that to not be confused with this one? Unless I'm missing something.

Reply:
Yeah. Maybe the other line should be reworded. I’ll take a look.